laravel-app/app/Http/Controllers/FollowController.php

```diff
@@ -29,10 +29,10 @@ public function update(User $user): RedirectResponse
 
         $success = ['success' => 'Successfully followed ' . $user->username];
 
-        return redirect()->back()->with($success);
+        return response()->json(['success' => $success]);
```

Comment:
Try to set a standard for your responses. Research related studies regarding construction of APIs.

e.g.
`return response()->json([
  'status' => 0,
  'message' => "Success. <some description>",
  'data' => [],
]);`